api/common/life.go

```diff
@@ -11,14 +11,29 @@ import (
 	"github.com/juju/juju/apiserver/params"
 )
 
-// Life requests the life cycle of the given entity from the given
+// Life requests the life cycle of the given entities from the given
 // server-side API facade via the given caller.
-func Life(caller base.FacadeCaller, tag names.Tag) (params.Life, error) {
+func Life(caller base.FacadeCaller, tags []names.Tag) (params.LifeResults, error) {
```

Comment:
The return type here should be []LifeResult
ie result.Results

Reply:
updated.